man/systemd.system-credentials.xml

```diff
@@ -471,6 +471,33 @@
           <xi:include href="version-info.xml" xpointer="v257"/>
         </listitem>
       </varlistentry>
+
+      <varlistentry>
+        <term><varname>userdb.user.*</varname></term>
+        <term><varname>userdb.group.*</varname></term>
+
+        <listitem>
+          <para>Configure JSON user and group records. Read by
+          <filename>systemd-userdb-load-credentials.service</filename>, which invokes
+          <command>userdbctl load-credentials</command>. These credentials directly translate to
+          matching
+          <ulink url="https://systemd.io/USER_RECORD">JSON User Record</ulink> and
+          <ulink url="https://systemd.io/GROUP_RECORD">JSON Group Record</ulink> records. Example: the
+          contents of a credential
+          <filename>userdb.user.foobar</filename> will be copied into a file
+          <filename>/run/userdb/foobar.user</filename>, and
+          <filename>userdb.group.foobar</filename> will be copied into a file
+          <filename>/run/userdb/foobar.group</filename>. Symlinks for the uid/gid will also be created in
+          <filename>/run/userdb</filename>, as well as the corresponding<filename>.membership</filename>
+          files. See
+          <citerefentry><refentrytitle>systemd-userdb</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
+          <citerefentry><refentrytitle>nss-systemd</refentrytitle><manvolnum>8</manvolnum></citerefentry>, and
+          <citerefentry><refentrytitle>userdbctl</refentrytitle><manvolnum>1</manvolnum></citerefentry>
+          for details.</para>
```

Comment:
maybe emphasize that these users/groups are not created persistently, i.e. in /run/ only

(though I have the suspicion, people will sooner or later want them persistently too, i.e in /etc).

Reply:
also, maybe emphasize that this does not touch /etc/passwd or /etc/group